tests/values_test.py

```diff
@@ -0,0 +1,18 @@
+from common import *
+
+
+def test_values(ds_local):
+    ds = ds_local
+
+    np.testing.assert_array_equal(ds['x'].values, np.array(ds.evaluate('x')).T)
+    np.testing.assert_array_equal(ds['name'].values, np.array(ds.evaluate('name')).T)
+
```

Comment:
This already works right, and no need to transpose

Reply:
Yeah, sorry, this test was written later, and I was just copy pasting code. I will improve it in the next iteration.